x/confidentialtransfers/types/params.go

```diff
@@ -11,7 +13,10 @@ func ParamKeyTable() paramtypes.KeyTable {
 
 // DefaultParams default confidential transfers module parameters.
 func DefaultParams() Params {
-	return Params{}
+	return Params{
```

Comment:
Looks like we might be missing some tests

Reply:
Do you think we need tests for DefaultParams and ParamSetPairs? They seem to just be returning objects

Reply:
Added tests for other methods here